frontend/src/metabase/admin/settings/components/Email/CloudSMTPConnectionCard.module.css

```diff
@@ -0,0 +1,23 @@
+.root {
+  background-color: var(--mb-color-bg-white);
+  border: 1px solid var(--mb-color-border);
+  border-radius: var(--mantine-radius-md);
+}
+
+.RadioCardRoot {
+  :hover {
+    background-color: var(--mb-color-bg-light);
+
+    .label {
+      color: var(--mb-color-brand);
+    }
+  }
+}
+
+.label {
+  font-weight: bold;
+}
+
+.indicator {
+  border: 1px solid gray;
```

Comment:
I need to change this colour, but the bigger question is why it's not displaying by default. Something is happening with the Mantine component styles not displaying correctly. I copied code directly from the Mantine example and it doesn't work